src/ServiceManagement/RemoteApp/Commands.RemoteApp/UserDisk/RemoveAzureRemoteAppUserDisk.cs

```diff
@@ -0,0 +1,49 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure;
+using Microsoft.WindowsAzure.Management.RemoteApp;
+using Microsoft.WindowsAzure.Management.RemoteApp.Models;
+using System.Management.Automation;
+
+namespace Microsoft.WindowsAzure.Management.RemoteApp.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRemoteAppUserDisk", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.High)]
```

Comment:
Hi Mark,

Yes, we are sure.

Thanks
Lei

From: Mark Cowlishaw [mailto:notifications@github.com]
Sent: Monday, May 9, 2016 3:57 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Lei He <hele@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-powershell] [UVHD Tenant Admin] Add admin cmdlet to delete and copy UVHD (#2229)


In src/ServiceManagement/RemoteApp/Commands.RemoteApp/UserDisk/RemoveAzureRemoteAppUserDisk.cs<https://github.com/Azure/azure-powershell/pull/2229#discussion_r62587217>:

> +// http://www.apache.org/licenses/LICENSE-2.0

> +// Unless required by applicable law or agreed to in writing, software

> +// distributed under the License is distributed on an "AS IS" BASIS,

> +// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.

> +// See the License for the specific language governing permissions and

> +// limitations under the License.

> +// ----------------------------------------------------------------------------------

> +

> +using Microsoft.Azure;

> +using Microsoft.WindowsAzure.Management.RemoteApp;

> +using Microsoft.WindowsAzure.Management.RemoteApp.Models;

> +using System.Management.Automation;

> +

> +namespace Microsoft.WindowsAzure.Management.RemoteApp.Cmdlets

> +{

> +    [Cmdlet(VerbsCommon.Remove, "AzureRemoteAppUserDisk", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.High)]

Are you sure you want to use ConfirmImpact.High? This means user would have to use -Confirm:$false on the command line in order to remove a disk in a script.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/2229/files/5f212ad5812b6990ab9a342b13824379dbf93c3a#r62587217>